src/draw/handler/Draw.Circle.js

```diff
@@ -15,7 +15,8 @@ L.Draw.Circle = L.Draw.SimpleShape.extend({
 			clickable: true
 		},
 		showRadius: true,
-		metric: true // Whether to use the metric meaurement system or imperial
+		metric: true, // Whether to use the metric meaurement system or imperial
+		toolbarIcon: { className: 'leaflet-draw-draw-circle', tooltip: 'Draw a circle.' }
```

Comment:
Fixed in 6ef4ea7. Thanks for catching this - I had forgotten about Leaflet.draw's localization support :).